src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -92,7 +92,7 @@ public void deleteWithId() throws Exception {
               ReadType.KEY,
               objectMapper);
 
-      DeleteOperation operation = new DeleteOperation(commandContext, findOperation);
+      DeleteOperation operation = new DeleteOperation(commandContext, findOperation, 1);
```

Comment:
Are these tests for `deleteOne()` or... ?

Reply:
DeleteOperation is used by both DeleteOne and DeleteMany, The existing deleteOne test cases I have changed to pass the limit to 1, similar to how its passed from DeleteOneCommandResolver class.